src/common/middleware/index.js

```diff
@@ -0,0 +1,10 @@
+export const analytics = () => next => action => {
```

Comment:
Can we have this one more readable?
I kinda know that it's a chain of functions that return functions, but it really hard to read and I don't even try to understand what it does.